corehq/apps/hqadmin/management/commands/static_analysis.py

```diff
@@ -81,6 +82,24 @@ def show_custom_modules(self):
         self.logger.log("commcare.static_analysis.custom_module_count", custom_module_count)
         self.logger.log("commcare.static_analysis.custom_domain_count", custom_domain_count)
 
+    def show_js_dependencies(self):
+        proc = subprocess.Popen(["./scripts/codechecks/hqDefine.sh", "static-analysis"], stdout=subprocess.PIPE)
+        output = proc.communicate()[0].strip().decode("utf-8")
+        (hqdefine_todo, hqdefine_done, requirejs_todo, requirejs_done) = output.split(" ")
+
+        self.logger.log("commcare.static_analysis.hqdefine_file_count", int(hqdefine_todo), tags={
+            'status': 'todo',
+        })
+        self.logger.log("commcare.static_analysis.hqdefine_file_count", int(hqdefine_done), tags={
+            'status': 'done',
+        })
+        self.logger.log("commcare.static_analysis.requirejs_file_count", int(requirejs_todo), tags={
+            'status': 'todo',
+        })
+        self.logger.log("commcare.static_analysis.requirejs_file_count", int(requirejs_done), tags={
+            'status': 'done',
+        })
```

Comment:
`tags` should be a list of strings like `"environment:travis"`.  I did that because that's the format anticipated by the datadog API library, but I agree it's a little nonsensical and would be open to switching to this format.

Reply:
oops, fixed in bf4f5a7f29f - I think either format is fine.